src/IFrameNavigator.ts

```diff
@@ -347,11 +352,22 @@ export default class IFrameNavigator implements Navigator {
     private handleResize(): void {
         const selectedView = this.settings.getSelectedView();
         const oldPosition = selectedView.getCurrentPosition();
-        selectedView.goToPosition(oldPosition);
-    }
 
-    private async handleScroll(): Promise<void> {
-        return this.saveCurrentReadingPosition();
+        const fontSize = this.settings.getSelectedFontSize();
+        this.iframe.contentDocument.body.style.fontSize = fontSize;
+        this.iframe.contentDocument.body.style.lineHeight = "1.5";
+
+        const sideMargin = parseInt(fontSize.slice(0, -2)) * 2;
```

Comment:
This seems to work, but it also seems that it assumes font sizes are set in terms of pixels. If this is the only place that assumption is made, it might be better to not depend on it so font sizes can also be given in terms of percentages or other metrics. I do not think the margin necessarily needs to change with the font size. (Ideally, it would be independently adjustable.)